apple/RNSVGRenderable.mm

```diff
@@ -698,4 +628,48 @@ - (void)resetProperties
   self.merging = false;
 }
 
+static CIImage *transparentImage()
```

Comment:
Can we move those to some `utils` directory so this file does not grow too much? Also you have some static fields here and also at the top of the file.